content/intro-to-storybook/angular/en/get-started.md

```diff
@@ -61,4 +297,14 @@ If you want to modify the styling, the source LESS files are provided in the Git
 
 ## Add assets
 
-We also need to add the font and icon [directories](https://github.com/chromaui/learnstorybook-code/tree/master/public) to the `assets/` folder. After adding styling and assets, the app will render a bit strangely. That’s OK. We aren’t working on the app right now. We’re starting off with building our first component!
+Add the font and icon directories by downloading them to your computer and dropping them into your repository.
+
+```bash
+svn export https://github.com/chromaui/learnstorybook-code/branches/master/public/icon assets/icon
+svn export https://github.com/chromaui/learnstorybook-code/branches/master/public/font assets/font
+```
+
+<div class="aside">
+<p>We’ve used <code>svn</code> (Subversion) to easily download a folder of files from GitHub. If you don’t have subversion installed or want to just do it manually, you can grab the folders directly <a href="https://github.com/chromaui/learnstorybook-code/tree/master/public">here</a>.</p></div>
```

Comment:
I would place this part before the commands to make it more accessible : explain then act.
And be more explicit about what users have to do once files are grabbed (ie copy them into the assets folder)

Reply:
@geromegrignon that section needs a bit of work, i've been thinking about this not only for this section but for the remainder of the documentation. Probably a better wording for this would be (posting it as i have it on my end):
```markdown

## Add assets

To match the intended design, you'll need to download both the font and icon directories and place its contents inside your `assets` folder.
```

```html
<div class="aside">
<p>We’ve used <code>svn</code> (Subversion) to easily download a folder of files from GitHub. If you don’t have subversion installed or want to just do it manually, you can grab the folders directly <a href="https://github.com/chromaui/learnstorybook-code/tree/master/public">here</a>.</p></div>
```

```bash
svn export https://github.com/chromaui/learnstorybook-code/branches/master/public/icon assets/icon
svn export https://github.com/chromaui/learnstorybook-code/branches/master/public/font assets/font
```

```markdown
After adding styling and assets, the app will render a bit strangely. That’s OK. We aren’t working on the app right now. We’re starting off with building our first component!
```

Do you agree with this? also @tmeasday and @domyen , do you both agree with the change?